docs/terms/_batches_and_batch_requests.mdx

```diff
@@ -29,51 +29,51 @@ as the time the batch was constructed, or hash of an in-memory DataFrame.
+    * A BatchRequest can include `data_connector_query` params with values relative to the latest Batch (e.g. the "latest" slice).
+      Conceptually, this enables "fetch the latest Batch" behavior. It is the key thing that differentiates
+      a BatchRequest, which does NOT necessarily uniquely identify the Batch(es) to be fetched, from a
       BatchDefinition.
     * The BatchRequest can also include a section called `batch_spec_passthrough` to make it easy to directly
-      communicate parameters to a specific ExecutionEngine.
-    * When resolved, the `BatchRequest` may point to many `BatchDefinitions` and Batches.
-
-* A BatchRequest is the entrypoint to getting a `Batch` from a Datasource, using
-  the `get_batch_list_from_batch_request()` method:
+      communicate parameters to a specific Execution Engine.
+    * When resolved, the BatchRequest may point to many BatchDefinitions and Batches.
+    * BatchRequests can be defined as dictionaries, or by instantiating a BatchRequest object.
+    
+    ```python
+    batch_request = {
+        "datasource": "myds",
+        "data_connector": "pipeline",
+        "in_memory_dataset": df,
+        "data_connector_query" : {
+          "batch_filter_parameters" : {
+              "airflow_run_id": my_run_id,
+              "other_key": my_other_key
+          }
+          "custom_filter_function": my_filter_fn,
+          "limit": 10,
+          "index": Optional[Union[int, list, tuple, slice, str]],  # examples: 0; "-1"; [3:7]; "[2:4]"
+        },
+        "sampling": {
+            "limit": 1000,
+            "sample": 0.1
+        }
+    }
+    ```
+    
+2. The Datasource finds the Data Connector indicated by the BatchRequest, and uses it to obtain a BatchDefinition list.
 
 ```python
-DataSource.get_batch_list_from_batch_request(batch_request={
-    "datasource": "myds",
-    "data_connector": "pipeline",
-    "in_memory_dataset": df,
-    "data_connector_query" : {
-      "batch_filter_parameters" : {
-          "airflow_run_id": my_run_id,
-          "other_key": my_other_key
-      }
-      "custom_filter_function": my_filter_fn,
-      "limit": 10,
-      "index": Optional[Union[int, list, tuple, slice, str]],  # examples: 0; "-1"; [3:7]; "[2:4]"
-    },
-    "sampling": {
-        "limit": 1000,
-        "sample": 0.1
-    }
-})
+DataSource.get_batch_list_from_batch_request(batch_request=batch_request)
 ```
-2. `BatchDefinition`
 
-* A `BatchDefinition` resolves any ambiguity in `BatchRequest` to uniquely identify a single `Batch` to be
-  fetched.  `BatchDefinition` is Datasource- and ExecutionEngine-agnostic. That means that its parameters may depend on
-  the configuration of the `Datasource`, but they do not otherwise depend on the specific `DataConnector` type (e.g.
-  filesystem, SQL, etc.) or `ExecutionEngine` being used to instantiate `Batches`.
+* A BatchDefinition resolves any ambiguity in BatchRequest to uniquely identify a single Batch to be
```

Comment:
I wonder if these details are necessary, since (at least in the Batch technical term page) they are already included above